packages/@sanity/form-builder/src/inputs/ImageInput/SelectAsset.js

```diff
@@ -46,6 +52,10 @@ export default class SelectAsset extends React.Component<Props, State> {
     })
   }
 
+  handleUpdate = () => {
```

Comment:
This should rather be `handleDeleteFinished`, shouldn't it?

Reply:
👍